src/main.rs

```diff
@@ -246,17 +225,59 @@ fn migrate(data_string: String) -> Result<String, anyhow::Error> {
     Ok(output_path_str.to_string())
 }
 
-#[launch]
-fn rocket() -> Rocket<Build> {
+async fn rm_file_after_expiration(database: &Arc<Mutex<Connection>>) -> Result<(), anyhow::Error> {
+    let database = database.lock().await;
+    let mut stmt = database.prepare(format!("SELECT * FROM {}", TABLE_NAME).as_str())?;
+    let rows = stmt.query([])?;
+    let rows = rows.mapped(|row| {
+        Ok(Entry {
+            uuid: row.get(0)?,
+            file_path: row.get(1)?,
+            creation_time: row.get(2)?,
+        })
+    });
+    for row in rows {
+        let row = row?;
+        let time_passed_since_creation = chrono::Local::now().time()
+            - chrono::DateTime::parse_from_str(row.creation_time.as_str(), TIME_FORMAT)?.time();
+        let t_delta = match time_passed_since_creation
+            .checked_sub(&chrono::TimeDelta::new(FILE_EXPIRATION_IN_SEC, 0).unwrap())
+        {
+            Some(t_delta) => t_delta,
+            None => chrono::TimeDelta::zero(),
+        };
```

Comment:
Maybe I should have said this earlier, but I'm wondering if a nicely formatted timestamp in the DB really is worth the whole hassle here. Normally I would have not used the `Chrono` crate here, but instead just save it as unix epoch timestamp, which would make this calculation a whole lot easier. But I'll leave it up to you.